Samples/ConsoleREPL/ConsoleREPL.cs

```diff
@@ -141,7 +141,7 @@ static string PrintResult(object value)
                     separator = "";
                     foreach (var row in table.Rows)
                     {
-                        resultString += separator + PrintResult(row.Value);
+                        resultString += separator + PrintResult(row.IsValue ? row.Value : row.Blank);

```

Comment:
This doesn't compile for me. It looks like these types don't match and that's not allowed in C# 8.0 which is what this project uses. Did you upgrade your project to .NET5 or 6 and that's why it works for you?
You could just cast these to "object" to ensure the types match which seems to compile fine.